infrastructure/azuredeploy.bicep

```diff
@@ -0,0 +1,46 @@
+targetScope = 'subscription'
+
+@allowed([ 'dev', 'prd' ])
+param environment string
+
+param targetBranch string
+
+var location = 'West Europe'
+
+var organizationPrefix = 'aio'
+var sharedResourcesAbbreviation = 'shr'
+var webAppResourcesAbbreviation = 'app'
+
+resource sharedRg 'Microsoft.Resources/resourceGroups@2022-09-01' = {
```

Comment:
We already create the shared resource group in the azure repository. This should not be a task of Shifty